evaluation.md

```diff
@@ -4,10 +4,13 @@
 
 _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
+Because this was an internal DSL and due to how the given libraries were implmented, I could not use keywords like if and else so I had to instead use beginning states like ifFacingEast, which maintains the conditional statement resemblance I was seeking. Additionally, I combined the statements for open and blocked as well as those for move and face because I was unable to find out how to make each of these occur separately then combine them post being called.
 
 **On a scale of 1–10 (where 10 is "a lot"), how much did you have to change your syntax?**
+I had to change my syntax about a 7 on a scale of 1-10. The language is much more verbose and hence much less resembles a programming language as I had originally desired.
```

Comment:
I agree that it changed a lot, but I don't dislike all of your changes - I guess I wonder if you move and face the same direction all the time, why you need to include that both in the name. Could it be less verbose if you just said "move north" or something? (lines 97 in Piconot.scala)

Reply:
Like, could you have renamed some of those functions to feel less cumbersome and more like a PL.